targets/TARGET_Freescale/TARGET_K20XX/TARGET_TEENSY3_1/device/system_MK20DX256.c

```diff
@@ -25,7 +27,7 @@
 **
 ** ###################################################################
 */
-
+ 
```

Comment:
Can you please remove these spaces from this commit?

Reply:
Changes made as requested